src/test/unit_tests/jest_tests/test_config_fs.test.js

```diff
@@ -22,9 +22,10 @@ describe('adjust_bucket_with_schema_updates', () => {
     });
 
     it('should return bucket without deprecated properties (system_owner)', () => {
```

Comment:
Please update the `it`
```suggestion
    it('should return bucket without deprecated properties', () => {
```